### PR TITLE
Upstream/985 combine multiple response projections

### PR DESCRIPTION
**Pull request by jxnu-liguobin**
---

### Description

Related to #ISSUE_NUMBER

---

Changes were made to:
- [ ] Codegen library - Java
- [ ] Codegen library - Kotlin
- [ ] Codegen library - Scala
- [ ] Maven plugin
- [ ] Gradle plugin
- [ ] SBT plugin

### CODE REVIEW

**Comment by jxnu-liguobin** [comment]
In order to verify major changes, I put Scala and kotlin files into the test module to ensure that they can be compiled, but at present, I need to manually execute GraphQLCodegenVerifyCompileTest.java

File: build.gradle [hunk 1]
```diff
@@ -7,6 +7,7 @@ plugins {
     id "signing"
     id "maven-publish"
     id "org.sonarqube" version "3.2.0"
+    id 'org.jetbrains.kotlin.jvm' version '1.8.20-Beta'
```
[the remark this answers is not in the document]
Comment (kobylynskyi) [reply build.gradle:10]: @jxnu-liguobin I would like not to add kotlin code and dependencies to the main project. instead, please add this as part of example kotlin plugin which we already have.

**Review by kobylynskyi — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (jxnu-liguobin) [reply build.gradle:10]: Yes. We can update the sample synchronously when major changes occur.

[the remark this answers is not in the document]
Comment (jxnu-liguobin) [reply build.gradle:10]: The template changes in this PR have been verified, closed it

**Pull request closed without merging**